ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -692,3 +721,132 @@ func validateTaskNetworkConfig(taskNetworkConfig *state.TaskNetworkConfig) error
 
 	return nil
 }
+
+// checkPacketLossFault checks if there's existing network-packet-loss fault running.
+func (h *FaultHandler) checkPacketLossFault(taskMetadata *state.TaskResponse, request types.NetworkPacketLossRequest) (bool, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
```

Comment:
can we do one more thing that when the it timed out, we should cancel the command execution?

Reply:
We use `CommandContext()` from `os/exec`, so the command execution will be cancelled when it times out.